woocommerce/woocommerce-android
# Add flow: External products

Fixes #2906. This PR implements the **add new product functionality** for external products.

**To test:**
1. Go to Products
2. Tap on Add product FAB
3. Notice that External products option is enabled
4. Tap on the external product
5. Notice that the product type is set as External product and all the properties should match the design
6. Fill out the properties and publish the product
7. After saving's done, tap on the back button and notice that the product now appears in the product list
8. Open the product again and verify all the data is present
